src/features/base-layout/components/navbar/dropdown.tsx

```diff
@@ -3,17 +3,23 @@ import { useOutsideClick } from '@/app/hooks';
 
 export interface DropdownProps {
   onMouseLeave: () => void;
+  onMouseEnter: () => void;
   isDropdownOpen: boolean;
   handleClose: () => void;
```

Comment:
```suggestion
   isOpen: boolean;
   onClose: () => void;
```
Consistent Prop Naming: To avoid any confusion, you may want to consider standardizing the naming of your properties. You use handleClose and onMouseLeave, onMouseEnter. All these are event handlers, yet their naming is inconsistent. It would be more consistent if you use the prefix 'on' or 'handle' for all event handler props. For example, handleClose could be renamed to onClose for consistency.

Reply:
Thank you for the advice. Changed